libs/etterlatte-behandling-model/src/main/kotlin/no/nav/etterlatte/libs/common/behandling/Etterbetalingmodell.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.etterlatte.libs.common.behandling
+
+import java.time.LocalDate
+import java.util.UUID
+
+data class Etterbetalingmodell(
```

Comment:
Ny navnekonvensjon?

Reply:
Skulle slette den gamle Etterbetaling-implementasjonen, så brukte dette namnet midlertidig. Så gløymte eg å slette 🙈  Fiksa i to nyaste commits no.